core/src/test/java/io/opencensus/stats/ViewTest.java

```diff
@@ -13,118 +13,78 @@
 
 package io.opencensus.stats;
 
-import static com.google.common.truth.Truth.assertThat;
-import static org.junit.Assert.assertTrue;
-
 import com.google.common.testing.EqualsTester;
 import io.opencensus.common.Duration;
-import io.opencensus.common.Function;
-import io.opencensus.stats.View.DistributionView;
-import io.opencensus.stats.View.IntervalView;
-import java.util.Arrays;
-import java.util.List;
+import io.opencensus.stats.Aggregation.Count;
+import io.opencensus.stats.Aggregation.Sum;
+import io.opencensus.stats.View.Window.CumulativeWindow;
+import io.opencensus.stats.View.Window.IntervalWindow;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
+import java.util.Arrays;
+import java.util.List;
+
+import static com.google.common.truth.Truth.assertThat;
+
 /** Tests for {@link View} */
 @RunWith(JUnit4.class)
 public final class ViewTest {
   @Test
   public void testDistributionView() {
-    DistributionAggregation dAggr = DistributionAggregation.create();
-    final View view = DistributionView.create(
-        name, description, measure, dAggr, keys);
-
+    final View view = View.create(
+        name, description, measure, aggregations, keys, CumulativeWindow.create());
     assertThat(view.getName()).isEqualTo(name);
     assertThat(view.getDescription()).isEqualTo(description);
     assertThat(view.getMeasure().getName()).isEqualTo(measure.getName());
+    assertThat(view.getAggregations()).isEqualTo(aggregations);
     assertThat(view.getColumns()).hasSize(2);
     assertThat(view.getColumns().get(0).toString()).isEqualTo("foo");
     assertThat(view.getColumns().get(1).toString()).isEqualTo("bar");
-    assertTrue(view.match(
-        new Function<DistributionView, Boolean> () {
-          @Override public Boolean apply(DistributionView dView) {
-            return dView == view;
-          }
-        },
-        new Function<IntervalView, Boolean> () {
-          @Override public Boolean apply(IntervalView iView) {
-            return false;
-          }
-        }));
+    assertThat(view.getWindow()).isEqualTo(RpcViewConstants.CUMULATIVE);
```

Comment:
Same here, this test shouldn't depend on the RPC constants, especially if changing their values would cause the test to fail.